# This document was autogenerated by the script camera_robot_calibration.py
# EDITING THIS FILE BY HAND IS NOT RECOMMENDED
# NOTE 1: This is the transformation between the camera coordinates and the robot reference system.
# More information: http://www.ensenso.com/manual/index.html?_cameras_byserialno_$stereoserial_link.htm

camera_right_robot:
  angle_axis: [3.132200431767123, 0.7203649106777076, 0.6935348755065331, -0.009152700171866278]
  child: right/base_link
  duration: 200.28700304
  grid_spacing: 15.94
  iterations: 64
  parent: camera_optical_frame
  rotation: [0.7203569674237653, 0.6935272280995705, -0.009152599247708797, 0.0046960936504344755]
  samples: 50
  seed:
    rotation: [0.7071067811865475, 0.7071067811865477, 4.3297802811774664e-17, 4.329780281177467e-17]
    translation: [0.605, -0.529, 1.206]
  translation: [0.5533346469851629, -0.6047394570270855, 1.2350182242637087]
reprojection_error: {pixels: 0.335648457619859}
